### PR TITLE
Add batch category function in backend

### PR DESCRIPTION
**Pull request by xillibit**
Pull Request for Issue # .

### CODE REVIEW

File: src/administrator/components/com_kunena/views/categories/view.html.php [hunk 1]
```diff
@@ -135,6 +137,18 @@ protected function setToolBarDefault()
 		JToolBarHelper::spacer();
 		$help_url  = 'https://www.kunena.org/docs/Sections,_Categories,_sub-Categories_(Part_1)';
 		JToolBarHelper::help('COM_KUNENA', false, $help_url);
+		
+		if (version_compare(JVERSION, '3', '>'))
```
Comment (810) [inline src/administrator/components/com_kunena/views/categories/view.html.php]: check can be removed

Comment (xillibit) [reply src/administrator/components/com_kunena/views/categories/view.html.php]: Done

**Comment by 810** [comment]
- [x] Move categories - Success
- [ ] Copy categories - Failed (it copied only one category)

test from older pr. Need to be retested again

**Comment by xillibit** [comment]
> Move categories - Success
>     Copy categories - Failed (it copied only one category)
> 
> test from older pr. Need to be retested again

I have updated it in order to make working copy

**Comment by 810** [comment]
Fatal error: Cannot redeclare KunenaForumCategoryHelper::getAlias() in \libraries\kunena\forum\category\helper.php on line 919

**Comment by 810** [comment]
is already on line 783

**Comment by 810** [comment]
- [x] Move catgories - Success
- [ ] Copy categories - Failed

Saving category 0 failed: Cannot save category: No name was provided!

**Comment by xillibit** [comment]
> Fatal error: Cannot redeclare KunenaForumCategoryHelper::getAlias() in \libraries\kunena\forum\category\helper.php on line 919

Fixed

**Comment by 810** [comment]
Saving category 0 failed: Cannot save category: No name was provided!

The category has been copied successfully

but the copy doesn't worked

**Comment by xillibit** [comment]
I have removed copy option because it doesn't work when you want to copy more than one category

**Comment by rich20** [comment]
> I have removed copy option because it doesn't work when you want to copy more than one category

Then should also be changed the text in modal "You can move or copy some categories by batch process." (views.ini line 1108)

**Pull request merged**